src/KubeOps/Operator/Comparing/ReferenceEqualityComparer.cs

```diff
@@ -4,7 +4,7 @@ namespace KubeOps.Operator.Comparing
 {
     internal class ReferenceEqualityComparer : EqualityComparer<object>
     {
-        public override bool Equals(object x, object y) => ReferenceEquals(x, y);
+        public override bool Equals(object? x, object? y) => ReferenceEquals(x, y);
```

Comment:
I only changed this because VS reported this as a warning and it went away making this change.